### PR TITLE
Apply reimbursement flag

### PR DESCRIPTION
**Pull request by ptbrowne**
- Applied reimbursement flag
- Updated tests to test new chips

### CODE REVIEW

File: src/ducks/transactions/TransactionRow.jsx [hunk 1]
```diff
@@ -236,18 +234,9 @@ class _RowMobile extends React.PureComponent {
...
-            <Img className={styles['bnk-transaction-mobile-action']}>
-              <TransactionActions
-                transaction={transaction}
-                onlyDefault
-                compact
-                menuPosition="right"
-              />
-            </Img>
-          )}
+          {false}
```
Comment (drazik) [inline src/ducks/transactions/TransactionRow.jsx]: It looks like we can remove this

File: src/ducks/transactions/actions/BillAction.jsx [hunk 2]
```diff
@@ -135,9 +135,7 @@ const action = {
   name,
   icon,
   match: async (transaction, actionProps) => {
-    if (flag('reimbursements.tag')) {
-      return false
-    }
+    return false
```
Comment (drazik) [inline src/ducks/transactions/actions/BillAction.jsx]: It means this action is not needed anymore, we can remove this file (AttachedDocsAction now shows the bills attached to a transaction)

File: src/ducks/transactions/TransactionRow.jsx (hunk 1 above)
Comment (ptbrowne) [inline src/ducks/transactions/TransactionRow.jsx]: Fixed

File: src/ducks/transactions/TransactionRow.jsx [hunk 3]
```diff
@@ -259,7 +248,7 @@ class _RowMobile extends React.PureComponent {
               styles.TransactionRowMobile__actions
             )}
           />
-        )}
+        }
```
Comment (ptbrowne) [inline src/ducks/transactions/TransactionRow.jsx]: Fixed

File: src/ducks/transactions/actions/HealthExpenseStatusAction.jsx [hunk 4]
```diff
@@ -169,11 +167,7 @@ const action = {
     return <Icon icon="hourglass" color={color} />
   },
   match: transaction => {
-    return (
-      isHealthExpense(transaction) &&
-      getVendors(transaction) &&
-      !flag('reimbursements.tag')
-    )
+    return false
```
Comment (drazik) [inline src/ducks/transactions/actions/HealthExpenseStatusAction.jsx]: This action is not used anymore, replaced by ReimbursementStatusAction. This file can be removed.

File: src/ducks/transactions/actions/BillAction.jsx (hunk 2 above)
Comment (ptbrowne) [reply src/ducks/transactions/actions/BillAction.jsx]: https://github.com/cozy/cozy-banks/issues/1754

File: src/ducks/transactions/TransactionRow.jsx (hunk 1 above)
Comment (ptbrowne) [reply src/ducks/transactions/TransactionRow.jsx]: Done.

File: src/ducks/transactions/actions/HealthExpenseStatusAction.jsx (hunk 4 above)
Comment (ptbrowne) [reply src/ducks/transactions/actions/HealthExpenseStatusAction.jsx]: https://github.com/cozy/cozy-banks/issues/1754

**Pull request merged**